content/zh-tw/style-guide/_index.md

```diff
@@ -49,8 +49,6 @@ status:
 category: 
 ---
 
-## 是什麼
-
 對該技術或概念的快速總結。
 
```

Comment:
Miss this change.

https://github.com/cncf/glossary/pull/2652/files#diff-f0035a3662384e762cfe06bd4661211988eae9f632b7e5ab740ef932a16df1acR82